docs/source/index.rst

```diff
@@ -6,15 +6,15 @@
 Welcome to the TStore documentation!
 ======================================
 
-The **TStore** is a Python package designed to make your life easier, whether you aim TODO.
-Our goal is to empower you to focus more on what you can discover and create with the data, rather than getting bogged down by the process of handling it.
+The **TStore** is a Python package designed to make your life easier when dealing with time series data. Our goal is to empower you to focus more on what you can discover and create with the data, rather than getting bogged down by the process of handling it.
 
-With our software, you can:
+With **TStore**, you can:
 
-1. **TODO**: Quickly search and download the data you need.
+1. Efficiently **store and read** *(probably almost) any form* time series data to disk. Thanks to its hierarchically-structured approach based on `Apache Parquet <https://parquet.apache.org>`__ and `GeoParquet <https://github.com/opengeospatial/geoparquet>`__, you can choose how samples and variables are stored and temporally partitioned.
 

```

Comment:
Read TSTORE data in the table format which is most suited to your analysis 
Multiple dataframe backend supported: pandas, dask, polars, pyarrow 
Enable distributed, lazy and and parallel processing 
Seamless integration with geopandas and xvec to manipulate spatial temporal database
Add something on irregular time series ... 
.....